react-native/src/TasksView.js

```diff
@@ -18,37 +17,37 @@ export function TasksView() {
   const result = useQuery('Task');
   const tasks = useMemo(() => result, [result]);
   const user = useUser();
-  // state value for toggable visibility of the 'CreateToDoPrompt' Model in the UI
-  const [createToDoOverlayVisible, setCreateToDoOverlayVisible] =
-    useState(false);
-
+  const [showNewTaskOverlay, setShowNewTaskOverlay] = useState(false);
+  
   // :state-uncomment-start: flexible-sync
   // useEffect(() => {
   //   // initialize the subscriptions
   //   const initSubscription = async () => {
   //     await realm.subscriptions.update(mutableSubs => {
-  //       mutableSubs.add(
-  //         realm.objects('Task').filtered(`owner_id == "${user.id}"`),
-  //       ); // subscribe to all Tasks of the logged in user
+  //       // subscribe to all Tasks of the logged in user
+  //       const ownTasks = realm
+  //         .objects('Task')
+  //         .filtered(`owner_id == "${user.id}"`);
```

Comment:
Is this what we decided on adding to the subs? Didn't we say we wanted to add a subscription to priority or is that not intended to be in this & will only be shown on the tutorial doc?